__tests__/index.js

```diff
@@ -2,43 +2,40 @@ const { createEvent } = require('@posthog/plugin-scaffold/test/utils.js')
 const { setupPlugin, add } = require('../index')
 
 test('test setting negative value for timeout in Twilio', async () => {
-    const response = await setupPlugin({
-        config: {
-            accountSID: 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
-            authToken: 'your_auth_token',
-            senderPhoneNumber: '+15555555555',
-            triggeringEventsAndNumber: 'event1:+15555555555,event2:+15555555555',
-            timeout: -1,
-        },
-        global: {},
-    })
-    expect(response).toEqual('timeout is not supported')
+    try {
+        const response = await setupPlugin({
+            config: {
+                accountSID: 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+                authToken: 'your_auth_token',
+                senderPhoneNumber: '+15555555555',
+                triggeringEventsAndNumber: 'event1:+15555555555,event2:+15555555555',
+                timeout: -1,
+            },
+            global: {},
+        })
+        // Fail test if above expression doesn't throw anything.
+      expect(true).toBe(false);
```

Comment:
this is not a great pattern.

Jest has utilities to check if a function throws.